CONTRIBUTING.md

```diff
@@ -80,9 +80,19 @@ This section should get you running with **Amplify CLI** and get you familiar wi
 
    # Windows
    yarn && yarn setup-dev-win
-   ## Must be run in Powershell
+   ## Preferably run in Git Bash
    ```
 
+   ### Additional Instructions for Windows Users:
+
+   Prior to running the `setup-dev` script:
+
+   1. Install the Visual C++ Build Environment by installing Visual Studio Community Edition. When selecting options, only 'Desktop Development for C++' needs to be added.
+   2. Open a terminal window/command prompt and run `npm config edit` and add or modify `msvs_version` setting to be your version of Visual Studio (most likely the 2022 edition, EX: `msvs_version=22`)
+   3. To avoid the build error 'MSB8040: Spectre-mitigated libraries are required for this project' open the Visual Studio installer, press the 'Modify' button for your version of Visual Studio, then navigate to the 'Individual Components' and search for 'Spectre'. Install options that look like "MSVC v143 - VS 2022 C++ x64/x86 Spectre-mitigated libs (Latest)", you should only need the x86-64 version, but can optionally install versions for ARM and ARM64/ARM64EC.
```

Comment:
```suggestion
   3. If you run into the build error 'MSB8040: Spectre-mitigated libraries are required for this project' open the Visual Studio installer, press the 'Modify' button for your version of Visual Studio, then navigate to the 'Individual Components' and search for 'Spectre'. Install options that look like "MSVC v143 - VS 2022 C++ x64/x86 Spectre-mitigated libs (Latest)", you should only need the x86-64 version, but can optionally install versions for ARM and ARM64/ARM64EC.
```

In case this goes away with new VS releases.